README.rst

```diff
@@ -62,38 +62,47 @@ them, please visit the following URLs:
 
 -  Announcements: http://lists.idyll.org/listinfo/khmer-announce
 
-We chat at https://gitter.im/dib-lab/khmer and the maintainers can be
-contacted at khmer-project@idyll.org.
+We have a slack chat that [you can join](LINK/EMAIL/JOIN) and the maintainers
```

Comment:
If I understand correctly, the slack chat is for "internal" lab communications. Of course we're pretty lax about how we define "internal" -- collaborators, colleagues, friends all welcome -- but giving access to the khmer channel requires membership in the dib-lab slack group, which I don't think we want to advertise as public.